EMMO/README.md

```diff
@@ -1,6 +1,49 @@
 # EMMO
 This [W3ID](https://w3id.org) provides a persistent URI namespace for the [Elementary Multiperspective Material Ontology (EMMO)](https://github.com/emmo-repo/EMMO/) and its domain ontologies.
 
+## Redirection Rules
+This section contains a general summary of the logic behind the redirection rules.  
+
+1. `https://w3id.org/emmo`
+   - If the user is accessing this from a browser: redirect to github pages documentation of this IRI. Otherwise, redirect to the `{DOMAIN}.ttl` file in the root of the master branch  
+
+2. `https://w3id.org/emmo#{NAME}`
+   - If the user is accessing this from a browser: redirect to github pages documentation of this IRI. Otherwise, redirect to the `{DOMAIN}.ttl` file in the root of the master branch  
+
+3. `https://w3id.org/emmo/`
+   - `{DOMAIN}.ttl` file in the root of the master branch  
+
+4. `https://w3id.org/emmo/{VERSION}`
+   - Given version of EMMO  
```

Comment:
Would it make sense to treat the same way as rule 1, except that it redirects to the given version?